module/VuFind/src/VuFind/ILS/Driver/Alma.php

```diff
@@ -865,66 +868,67 @@ public function getMyFines($patron)
      */
     public function getMyHolds($patron)
     {
-        $xml = $this->makeRequest(
-            '/users/' . $patron['id'] . '/requests',
-            ['request_type' => 'HOLD']
-        );
         $holdList = [];
-        foreach ($xml as $request) {
-            $lastInterestDate = $request->last_interest_date
-                ? $this->dateConverter->convertToDisplayDate(
-                    'Y-m-dT', (string)$request->last_interest_date
-                ) : null;
-            $available = (string)$request->request_status === 'On Hold Shelf';
-            $lastPickupDate = null;
-            if ($available) {
-                $lastPickupDate = $request->expiry_date
+        $offset = 0;
+        $totalCount = 1;
+        $allowCancelingAvailableRequests
+            = !isset($this->config['Holds']['allowCancelingAvailableRequests'])
+                || $this->config['Holds']['allowCancelingAvailableRequests'];
```

Comment:
This might be more readable as `$this->config['Holds']['allowCancelingAvailableRequests'] ?? true`.